### PR TITLE
add pipmonkey.html

### PR DESCRIPTION
**Pull request by pipmonkey**

### CODE REVIEW

**Comment by aaossa** [comment]
Hi @pipmonkey , this file seems to be ready, but you forgot to add your username to ‘human.txt‘. Can you add that file? Thank you :grin:

**Comment by pipmonkey** [comment]
oops, updated human.txt! thanks @aaossa

**Review by aaossa — APPROVED** [review]
Everything seems :ok_hand: Thanks @pipmonkey :raised_hands: :grin:

**Comment by aaossa** [comment]
Thanks for your PR @pipmonkey :grin: , you can see your page [here](http://muan.co/hello-world/pipmonkey.html) :confetti_ball: :tada: :confetti_ball:

**Comment by pipmonkey** [comment]
thanks @aaossa 💯  looks great!

File: pipmonkey.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+ <html>
+ <head>
+    <title>pipmonkey the codemonkey</title>
+  </head>
+  <body>
+    <h1>About</h1>
+    <p>Call me <em>pipmonkey</em>, brah.</p>
+    <a href="www.google.com">Go to the google homepage</a>
```
Comment (SubhrajyotiSen) [inline pipmonkey.html:9]: Can you please prefix the url with http://  😃 
The current url is relative to the page url

Comment (pipmonkey) [reply pipmonkey.html:9]: oops! thanks for pointing it out @SubhrajyotiSen 👍 createing new PR

**Pull request merged**